public/locations-list-map.js

```diff
@@ -583,3 +710,28 @@ function number_format(number, decimal_places, dec_seperator, thou_seperator) {
   return f[0] + dec_seperator + f[1];
 }
 
+
+
+/**
+ * Takes the form input and attempts to pull geocoded information from it. Once done, centers map on closest markers nearby.
+ */
+function loadSearchInMap() {
+  if (!autocomplete) {
+    console.error('autocomplete is not yet defined.');
+    return;
+  }
+
+  let place = autocomplete.getPlace();
```

Comment:
nit: lots of things seem to be "let" that could be "const." Can you make a quick pass to flip those if the variable is effectively a single-static-assignment?